### DISCUSSION TITLE
Dropdown: scroll event

### DISCUSSION THREAD

**Discussion by frasza** [primefaces/community#399] [PrimeVue] [answered]
Hello 👋

I am looking for a way to catch scroll event in Dropdown component. I am not using lazy virtual scroll or anything but do have a usecase where I would like to catch and react whenever I scroll to the bottom of the Dropdown list.
Any ideas if this is possible? If so, how? I tried with event listeners and other similar approaches without luck.

Thank you.

**Comment by TeoDar** [comment, marked as answer]
Perhaps the idea [behind this answer](https://github.com/orgs/primefaces/discussions/340#discussioncomment-7478048) will help you?

If not, please provide a link, example from StackBlitz.

**Comment by frasza** [comment]
@TeoDar Freaking beautiful. Works like a charm and gives so much insight in how it works. Thank you!
